### DISCUSSION TITLE
I downlaod the Video only in 852x480p

### DISCUSSION THREAD

**Discussion by KalliBread** [aclap-dev/video-downloadhelper#347] [Help and Q&A]
### Your question

Hello folks,

I would have a question. When I want to download videos via my web browser (Chrome), I get the messages in which qualities I want to download it. So I keep clicking 1080p. However, it is downloaded or saved in 480p. Do I have to change something in the settings? In the settings you can choose between 1080p, 720p and 480p for the stream video. This is how I know that there is also 1080p video in the stream. I also bought the full version.

Would anyone have an idea?

### Addon & CoApp version and details

Video Download in 1080p and not in 480p

**Comment by Mattmjs** [comment]
And where are you seeing this happen, have you got an example.

**Comment by KalliBread** [comment]
How do you mean it?

After i Dowload the Video-File, i Click with the Right Mouse --> Settings --> Details and i see ist on the Settings

![Screenshot 2024-04-01 213350](https://github.com/aclap-dev/video-downloadhelper/assets/165498057/b6ccc8c9-2ca3-4f14-98e5-ece73d3e0447)


When i click in the VideoDownloader Helper on Details on the File i can see that ist in 1080p available.

![Bild 1](https://github.com/aclap-dev/video-downloadhelper/assets/165498057/16ef4ab9-e51b-46c7-858f-820eec2e0cbf)
![Screenshot 2024-04-01 214730](https://github.com/aclap-dev/video-downloadhelper/assets/165498057/c518fcf4-e066-44bf-a895-85c2e4191854)

**Reply by Mattmjs** [reply]
Ok I see, I'd like to try myself but the video is private and not public. Am I correct ?
You've hidden some of the important information in your screenshot which could be useful. Other than that I don't know how else
you could be assisted on this. Maybe try in Firefox.

One thing you've left out is your OS and versions of the extension and companion app. I'm just another user.

**Comment by KalliBread** [comment]
Yes, that's right, the video is private and not public. To make it short. The videos are on a OneDrive server and can be viewed via MS Stream. Downloading the videos is not easily possible.

My operating system is Windows 11 and I use the Chrome web browser (latest version). I try it via Firefox browser and I had the same problem.

**Reply by Mattmjs** [reply]
What happens if you try the other 720p & 480p versions. What results do you get on those.

**Reply by KalliBread** [reply]
the same result :/. The file will be in 480p

**Reply by Mattmjs** [reply]
Unusual case, if I could look at that one I'd look a little deeper to see what's going on. Maybe It might be different using
ffmpeg directly maybe not.

**Reply by KalliBread** [reply]
Mhm... what can I post so that you have a better insight?

**Reply by KalliBread** [reply]
Whatever comes to mind. I downloaded a video from YouTube using Firefox. This was actually downloaded in 1080p.

**Reply by Mattmjs** [reply]
I could use that mpd url that was blacked out or does it contain private info. Would you be able to email it if can't post here ?

**Reply by KalliBread** [reply]
it Contain Private Info. if i can send you a private message would is be great.

**Reply by Mattmjs** [reply]
See my post above, I will edit it out once you've sent it.

**Reply by KalliBread** [reply]
Thanks for the help. I sent you an email.
